dev-vcs/git/git-9999.ebuild

```diff
@@ -558,6 +563,12 @@ src_install() {
 		popd &>/dev/null || die
 	fi
 
+	if use netrc ; then
```

Comment:
I wonder if we should just install it "unconditionally" with USE=perl. The extra files are tiny and it's probably more bother to make it conditional than it is to not.

What do you reckon?